scripts/xonsh

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env python -u
-from xonsh.main import main
-main()
+#!/usr/bin/env sh
```

Comment:
I am not sure how I feel about this. This line means that to start xonsh requires a posix compatible shell, which is the whole thing we are trying to avoid.  I think this version could make an interesting alternative script (maybe called `xonshell`?) or something for people who really need it.